definitions/marts/bigquery_marts/npq/npq_enrolments.sqlx

```diff
@@ -1,18 +1,18 @@
 config {
     type: "table",
     assertions: {
-        uniqueKey: ["application_ecf_id"]
+        uniqueKey: ["application_id"]

```

Comment:
Is this identical to the version I approved before I went on leave? If so I'll approve it, otherwise can you point me to any changes?

Reply:
It is the same as the V2 file you approved previously.